CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Current development version
 
+- ** BREAKING **: Requires OrientDB 2.2.28+, since it depends on two OrientDB bugs being fixed: [bug 1](https://github.com/orientechnologies/orientdb/issues/7225) [bug 2](https://github.com/orientechnologies/orientdb/issues/7754)
```

Comment:
Do you mean : `**BREAKING**` ? (If you want the italics, that is).

Reply:
Hmm I guess this broke the formatting -- I'll fix it, good catch.